.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 ## Summary
 <!-- Briefly describe the changes and the purpose of the PR. -->
 
+> Note: Prefer semantic design tokens/utilities (e.g., text-text, bg-surface, border-border). Avoid raw colors and ad-hoc Tailwind color scales.
```

Comment:
I don't think this is something that should be added to `.github/pull_request_template.md`.